source/npm/qsharp/ux/circuit-vis/formatters/gateFormatter.ts

```diff
@@ -184,28 +204,28 @@ const _gatePosition = (
   const maxY = Math.max(...ys);
   const minY = Math.min(...ys);
 
-  let x1: number, y1: number, x2: number, y2: number;
+  let x1: number, y1: number, w: number, h: number;
 
   switch (type) {
     case GateType.Group: {
       const padding = groupBoxPadding - nestedDepth * nestedGroupPadding;
 
       x1 = x - 2 * padding;
       y1 = minY - gateHeight / 2 - padding;
-      x2 = width + 2 * padding;
-      y2 = maxY + +gateHeight / 2 + padding - (minY - gateHeight / 2 - padding);
+      w = width + 2 * padding;
+      h = maxY - minY + gateHeight + 2 * padding;
```

Comment:
I can see the equation from before was extra confusing, I'm still confused why `maxY - minY` is needed. Shouldn't height just be `gateHeight + 2 * padding`?